### PR TITLE
refactor Profile using Item

### PR DESCRIPTION
**Pull request by Tuxosaurus**
profile_default
profile_medium

### CODE REVIEW

File: src/profile/index.tsx [hunk 1]
```diff
@@ -1,58 +1,61 @@
...
-      {action &&
-        <div>
-          <Button status={Button.STATUS.UNSTYLED} title={action}>
-            <ArrowIcon right />
-          </Button>
-        </div>
-      }
-      <style jsx>{style}</style>
-    </div>
+      chevron={!!isLink}
```
Comment (wakooka) [inline src/profile/index.tsx]: `{isLink}` instead because it's already a boolean

Comment (Tuxosaurus) [reply src/profile/index.tsx]: Fixed

File: src/profile/index.tsx [hunk 2]
```diff
@@ -1,58 +1,61 @@
...
+  picture,
+  alt,
+  isIdChecked,
+  isMedium,
+  isLink,
+  score,
+  ratings,
+  ratingsLabel,
+}: ProfileProps) => {
+  const getLeftBody = () => (
```
Comment (pith) [inline src/profile/index.tsx]: nit: doesn't need to be a function

Comment (Tuxosaurus) [reply src/profile/index.tsx]: Right, changed

**Pull request merged**